core/src/main/kotlin/com/google/container/tools/core/analytics/UsageTrackerManagerService.kt

```diff
@@ -18,32 +18,42 @@ package com.google.container.tools.core.analytics
 
 import com.google.common.annotations.VisibleForTesting
 import com.google.container.tools.core.properties.PluginPropertiesFileReader
+import com.intellij.ide.util.PropertiesComponent
 import com.intellij.openapi.application.ApplicationManager
 import com.intellij.openapi.components.ServiceManager
 
 /**
  * Application service that manages the status of usage tracking in the plugin.
  */
-class UsageTrackerManagerService {
+class UsageTrackerManagerService(
+    private val trackingPreferenceProperty: PropertiesComponent = PropertiesComponent.getInstance()
```

Comment:
It might be useful to add a short parameter style comment for this, i.e. how is this used in a few words.

Reply:
done